Content.Server/Singularity/EntitySystems/EventHorizonSystem.cs

```diff
@@ -131,6 +135,13 @@ public void ConsumeEntity(EntityUid hungry, EntityUid morsel, EventHorizonCompon
         }
 
         EntityManager.QueueDeleteEntity(morsel);
+
+        if (eventHorizon.InheritMomentum && TryComp<PhysicsComponent>(hungry, out var thisPhysics) && TryComp<PhysicsComponent>(morsel, out var otherPhysics))
+        {
+            var impulse = (otherPhysics.LinearVelocity - thisPhysics.LinearVelocity) * otherPhysics.FixturesMass;
```

Comment:
don't think it should, there's nowhere it would account for the other body's velocity
could add angular momentum, yes

Reply:
The singularities body should already be accounted for. It basically just +=s the applied momentum.

Reply:
Example: You have two singulos of equal mass. One is heading to the right at 1m/s and the other is heading left at 1m/s according to the rest frame. They collide.
What should happen: there is now one, stationary singulo of twice the mass.
What will happen here: Either the first eats the second and we wind up with the singulo moving to the left at 1/2 m/s or the second eats the first and it winds up moving to the right at 1/2 m/s. This is because this boosted the eaten singularity to 2m/s in its direction of travel.

Reply:
i think you might be missing something: consider what happens if 2 objects with equal velocities somehow collide, my code would work correctly here, but not having the substract would break

Reply:
i have now fixed this by adding a term to account for the mass change